### PR TITLE
fix(components): [select-v2] status-icon position

### LINKED ISSUE #7785: [Bug Report] [Component] el-form 的 status-icon 在 el-select-v2 内位置错乱
Bug Type: **`Component`**

## Environment

- Vue Version: `3.2.33`
- Element Plus Version: `2.2.0`
- Browser / OS: `'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.60 Safari/537.36'`
- Build Tool: `Vite`

### Related Component

- `el-form`
- `el-form-item`
- `el-select-v2`

### Reproduction Link

[Element Plus Playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtZm9ybVxuICAgIHJlZj1cInJ1bGVGb3JtUmVmXCJcbiAgICA6bW9kZWw9XCJydWxlRm9ybVwiXG4gICAgOnJ1bGVzPVwicnVsZXNcIlxuICAgIGxhYmVsLXdpZHRoPVwiMTIwcHhcIlxuICAgIGNsYXNzPVwiZGVtby1ydWxlRm9ybVwiXG4gICAgOnNpemU9XCJmb3JtU2l6ZVwiXG4gICAgc3RhdHVzLWljb25cbiAgPlxuICAgIDxlbC1mb3JtLWl0ZW0gbGFiZWw9XCJBY3Rpdml0eSB6b25lXCIgcHJvcD1cInJlZ2lvblwiPlxuICAgICAgPGVsLXNlbGVjdC12MiB2LW1vZGVsPVwicnVsZUZvcm0ucmVnaW9uXCIgOm9wdGlvbnM9XCJvcHRpb25zXCIgY2xlYXJhYmxlIHBsYWNlaG9sZGVyPVwiQWN0aXZpdHkgem9uZVwiPlxuICAgICAgPC9lbC1zZWxlY3QtdjI+XG4gICAgPC9lbC1mb3JtLWl0ZW0+XG4gIDwvZWwtZm9ybT5cbjwvdGVtcGxhdGU+XG5cbjxzY3JpcHQgbGFuZz1cInRzXCIgc2V0dXA+XG5pbXBvcnQgeyByZWFjdGl2ZSwgcmVmIH0gZnJvbSAndnVlJ1xuaW1wb3J0IHR5cGUgeyBGb3JtSW5zdGFuY2UsIEZvcm1SdWxlcyB9IGZyb20gJ2VsZW1lbnQtcGx1cydcblxuY29uc3QgZm9ybVNpemUgPSByZWYoJ2RlZmF1bHQnKVxuY29uc3QgcnVsZUZvcm1SZWYgPSByZWY8Rm9ybUluc3RhbmNlPigpXG5jb25zdCBydWxlRm9ybSA9IHJlYWN0aXZlKHtcbiAgcmVnaW9uOiAnJ1xufSlcblxuY29uc3Qgb3B0aW9ucyA9IFt7XG4gIGxhYmVsOiAnWm9uZSBvbmUnLFxuICB2YWx1ZTogJ3NoYW5naGFpJ1xufSwge1xuICBsYWJlbDogJ1pvbmUgdHdvJyxcbiAgdmFsdWU6ICdiZWlqaW5nJ1xufV1cblxuY29uc3QgcnVsZXMgPSByZWFjdGl2ZTxGb3JtUnVsZXM+KHtcbiAgcmVnaW9uOiBbXG4gICAge1xuICAgICAgcmVxdWlyZWQ6IHRydWUsXG4gICAgICBtZXNzYWdlOiAnUGxlYXNlIHNlbGVjdCBBY3Rpdml0eSB6b25lJyxcbiAgICAgIHRyaWdnZXI6ICdjaGFuZ2UnLFxuICAgIH0sXG4gIF1cbn0pXG5cbmNvbnN0IHN1Ym1pdEZvcm0gPSBhc3luYyAoZm9ybUVsOiBGb3JtSW5zdGFuY2UgfCB1bmRlZmluZWQpID0+IHtcbiAgaWYgKCFmb3JtRWwpIHJldHVyblxuICBhd2FpdCBmb3JtRWwudmFsaWRhdGUoKHZhbGlkLCBmaWVsZHMpID0+IHtcbiAgICBpZiAodmFsaWQpIHtcbiAgICAgIGNvbnNvbGUubG9nKCdzdWJtaXQhJylcbiAgICB9IGVsc2Uge1xuICAgICAgY29uc29sZS5sb2coJ2Vycm9yIHN1Ym1pdCEnLCBmaWVsZHMpXG4gICAgfVxuICB9KVxufVxuXG5jb25zdCByZXNldEZvcm0gPSAoZm9ybUVsOiBGb3JtSW5zdGFuY2UgfCB1bmRlZmluZWQpID0+IHtcbiAgaWYgKCFmb3JtRWwpIHJldHVyblxuICBmb3JtRWwucmVzZXRGaWVsZHMoKVxufVxuPC9zY3JpcHQ+XG4iLCJpbXBvcnRfbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7fVxufSIsIl9vIjp7fX0=)

### Steps to reproduce

1.打开Playground
2.点击下拉选择器并选中任意一项/取消选中任意一项

### What is Expected?

el-form 的 status-icon 垂直居中

### What is actually happening?

el-form 的 status-icon 没有垂直居中

### Additional comments
image
image

### PR DESCRIPTION
**Pull request by chenxch**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

fix #7785

### CODE REVIEW

**Comment by chenxch** [comment]
@YunYouJun Need your help to make sure this is reasonable.

**Comment by YunYouJun** [comment]
Can you show me a screenshot of your modification?
I think this is also good for future inspections.

**Comment by chenxch** [comment]
> Can you show me a screenshot of your modification? I think this is also good for future inspections.

OK, will handle this later.
image
image

**Comment by YunYouJun** [comment]
Where can i find it? 
image

**Comment by chenxch** [comment]
> Where can i find it? image

#7785

**Comment by YunYouJun** [comment]
Yes, I mean I don't seem to find this example in the documentation. How about adding it in?

**Comment by chenxch** [comment]
> 

This is an additional property of the `form`

**Comment by chenxch** [comment]
https://github.com/element-plus/element-plus/blob/50ade1267ec894302c0951b51497e3611cbb5c71/packages/components/input/src/input.vue#L100-L109

**Comment by chenxch** [comment]
Of course, we should add an example to it. This problem occurs on `select-v2` and does not appear in `select`. I compared and found that `select-v2` uses `input`, `select` uses `el-input`, and `el-input` wraps `el-input__suffix-inner` outside the `icon` to make It is normally centered.

File: packages/theme-chalk/src/select-v2.scss [hunk 1]
```diff
@@ -217,6 +217,10 @@ $input-inline-start: map.get($input-padding-horizontal, 'default') !default;
       #{getCssVarName('input-icon-color')},
       map.get($input, 'icon-color')
     );
+
+    & .#{$namespace}-input__icon {
+      height: inherit;
+    }
```
Comment (JeremyWuuuuu) [inline packages/theme-chalk/src/select-v2.scss:223]: **Suggestion:** remove `&` here since it behaves the same w/ or w/o this symbol.

Comment (chenxch) [reply packages/theme-chalk/src/select-v2.scss]: OK.

**Comment by YunYouJun** [comment]
Add a select v2 example to the form, and let's merge it.

**Comment by chenxch** [comment]
@YunYouJun  PTAL.

**Review by YunYouJun — APPROVED** [review]

**Pull request merged**